graal/backends/core/colic.py

```diff
@@ -170,17 +190,20 @@ class LicenseAnalyzer:
     """Class to analyse the content of files
 
     :param exec_path: path of the license analyzer executable
-    :param kind: the analyzer kind (e.g., NOMOS, SCANCODE)
+    :param kind: the analyzer kind (e.g., NOMOS, SCANCODE, SCANCODE_CLI)
     """
 
     def __init__(self, exec_path, kind=NOMOS):
+        self.kind = kind
         if kind == SCANCODE:
             self.analyzer = ScanCode(exec_path)
+        elif kind == SCANCODE_CLI:
+            self.analyzer = ScanCode(exec_path, cli=True)
         else:
             self.analyzer = Nomos(exec_path)
 
     def analyze(self, file_path):
-        """Analyze the content of a file using Nomos
+        """Analyze the content of a file using Nomos/Scancode
```

Comment:
@inishchith can you update the doc as done at: https://github.com/chaoss/grimoirelab-graal/pull/29/files#diff-88309ccaaf85d4fc1ec566c704a98b82R119 ?